src/common/raft/proto/raft_index.cuh

```diff
@@ -20,377 +20,230 @@
 #include <istream>
 #include <optional>
 #include <ostream>
-#include <type_traits>
```

Comment:
I believe all changes in this file are the result of applying the linters to CUDA code.